main.nf

```diff
@@ -295,6 +314,14 @@ process bowtie_miRBase_hairpin {
 }
 
 
+def rule_step5(file){
```

Comment:
Can we have a nicer name for this? `def mature_hairpin_publishDirs` or something. `rule_step5` is a bit non-descriptive, and the step numbers are easily changed.